src/catchup/CatchupWork.cpp

```diff
@@ -601,6 +619,13 @@ CatchupWork::runCatchupStep()
                 mBucketVerifyApplySeq = downloadApplyBuckets();
                 seq.push_back(mBucketVerifyApplySeq);
             }
+            // If there are no buckets to apply, still need to index BL before
+            // ledgers can be replayed
+            else if (mApp.getConfig().EXPERIMENTAL_BUCKETLIST_DB)
+            {
+                auto indexBuckets = std::make_shared<IndexBucketsWork>(mApp);
```

Comment:
shouldn't this be taken care of by indexing on startup?

Reply:
Yes, removed this block.